test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be added to the `.gitignore` see the requirement

Reply:
thanks

i have done that